app/app_internal_test.go

```diff
@@ -47,7 +47,7 @@ func Test_eth2Client(t *testing.T) {
 
 	ctx := context.Background()
 
-	client, err := eth2Client(ctx, srv.URL, 1)
+	client, err := eth2Client(ctx, srv.URL, 1, 1)
```

Comment:
Will/would this slow our test suite down a decent bit? Any reason not to have the default be like the user facing default?

Reply:
Nope, the integrated mock server is far faster than any BN out there. Also this test only has 1 validator, so it'll be alright.